src/components/InputField/InputField.stories.mdx

```diff
@@ -106,11 +106,11 @@ export default InputFieldExample;
 | tabIndex       | indicates the element can be focused, and participates in sequential keyboard navigation | `number`                                                                    |
 | autoComplete   | specifies whether an element should have autocomplete enabled                            | `string`                                                                    |
 | onBlur         | event when an element loses focus                                                        | `func`                                                                      |
-| onFocus        | event when the user sets focus on an element                                             | `number`                                                                    |
+| onFocus        | event when the user sets focus on an element                                             | `func`                                                                      |
```

Comment:
Hey, `func` is a wrong type name, please rename it to `function`
Thanks.

Reply:
Hey, used `func` since it was present in other places in the file too. Will rename it.